controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -28,6 +29,8 @@ def output_path(
         directory.mkdir(parents=True, exist_ok=True)
 
     p = directory / Path(f'{team_blue}_vs_{team_yellow}-{now_str}')
```

Comment:
I didn't touch that one for lack of understanding, I just stuck my output in where it looked plausible to override... Do you mean we should move the entire function down the file? Maybe we should consider doing that later and/or on master to avoid weird merge conflicts.

Reply:
I just wanted to make the function more readable.

```python
def output_path( 
    directory: Path,
    team_blue: str,
    team_yellow: str,
) -> PosixPath:
    now_str = datetime.utcnow().strftime('%Y%m%dT%H%M%S.%fZ')
    team_blue = team_blue.replace(' ', '_')
    team_yellow = team_yellow.replace(' ', '_')

    if automatic_mode:
        directory = Path('/out/')

    # Ensure the directory ecists
    if not directory.exists():
         directory.mkdir(parents=True, exist_ok=True)

    return directory / Path(f'{team_blue}_vs_{team_yellow}-{now_str}')
```

Reply:
Arguably, this should probably not happen inside the function at all -- we should make it so that the `directory` parameters that is passed to `output_path` is already `/out`-prefixed.

Reply:
Yeah, @mrshu is right, the directory should be provided in advance.

Reply:
Okay, so kill path generation and get output path from os.environ['video_path'] entirely?

Reply:
Not really, check line where `output_path` function is called and create `output_prefix` like this

```python
if automatic_mode:
    output_prefix = output_path(Path('/out'), TEAM_BLUE, TEAM_YELLOW)
else:
    output_prefix = output_path(Path('reflog'), TEAM_BLUE, TEAM_YELLOW)
```